### DISCUSSION TITLE
SVG Preview in CuratorColumn

### DISCUSSION THREAD

**Discussion by dave2084** [awcodes/filament-curator#518] [General]
Is is possible to show the preview of an SVG (rendered at the specified height) in filament table CuratorColumn?

Currently it just shows an SVG logo, which is itself an SVG!

**Comment by awcodes** [comment]
This is a bug, can you please submit an issue?

**Comment by dave2084** [comment]
Done: https://github.com/awcodes/filament-curator/issues/519
